schemas/SDMXStructureConstraint.xsd

```diff
@@ -480,42 +328,12 @@
 					<xs:element name="TimeRange" type="TimeRangeValueType"/>
 				</xs:choice>
 				<xs:attribute name="id" type="common:SingleNCNameIDType" use="required"/>
-			</xs:restriction>
-		</xs:complexContent>
-	</xs:complexType>
-
-	<xs:complexType name="ComponentValueSetType">
-		<xs:annotation>
-			<xs:documentation>ComponentValueSetType defines the structure for providing values for a data attributes, measures, or metadata attributes. If no values are provided, the component is implied to include/excluded from the region in which it is defined, with no regard to the value of the component. Note that for metadata attributes which occur within other metadata attributes, a nested identifier can be provided. For example, a value of CONTACT.ADDRESS.STREET refers to the metadata attribute with the identifier STREET which exists in the ADDRESS metadata attribute in the CONTACT metadata attribute, which is defined at the root of the report structure.</xs:documentation>
-		</xs:annotation>
-		<xs:complexContent>
-			<xs:restriction base="MemberSelectionType">

```

Comment:
In the past, we could include/exclude whole components (eg Attributes or Measure) using an empty Component element. This is not possible anymore.

Reply:
I did not realise this was a feature of the Constraint - to confirm - it is possible to reference an Attribute with exclude and no members at all and this means the Attribute is excluded for the whole collection (for the DSD/flow/provision)?

 If this is the case this behaviour it should be noted in the technical guide as I don't recall seeing it.

What does it mean if the attribute or meaure is an include and nothing is mentioned, I guess this is just meaningless?

Reply:
I have added minOccurs=0 back in

Reply:
For the empty Components in Constraints, this was my understanding (I found it also in my comments, but only in samples I have created during the 3.0 publication). You are right though, that it needs to go into the technical guide.